lib/signaling/v2/localparticipant.js

```diff
@@ -159,17 +160,21 @@ class LocalParticipantV2 extends LocalParticipantSignaling {
       // the "updated" event is emitted due to LocalTrackPublicationV2's
       // .isEnabled being toggled. We do not publish if it is fired due to the
       // LocalTrackPublicationV2's .sid being set.
-      if (isEnabled !== publication.isEnabled) {
+
+
+      // CHANGE THIS TO UPDATE ON TRACK PRIORITY UPDATES.
+      if (isEnabled !== publication.isEnabled || updatedPriority !== publication.updatedPriority) {
         this.didUpdate();
         isEnabled = publication.isEnabled;
+        updatedPriority = publication.updatedPriority;
       }
       if (!sid && publication.sid) {
         sid = publication.sid;
       }
-      if (sid && (updatedPriority !== publication.updatedPriority)) {
-        this.updateTrackPriority(publication.sid, publication.updatedPriority);
-        updatedPriority = publication.updatedPriority;
-      }
+      // if (sid && (updatedPriority !== publication.updatedPriority)) {
+      //   this.updateTrackPriority(publication.sid, publication.updatedPriority);
```

Comment:
Nitpick: Remove commented code

Reply:
@syerrapragada - I think you are looking at old commit in this change. I hav ealready removed these comments.  "copa" was just a placeholder I was using to tag all the changes needed to make RSP work :), It was removed in subsequent commits.